pkgs/by-name/an/antimatter-dimensions/package.nix

```
@@ -0,0 +1,71 @@
+{ buildNpmPackage
+, electron
+, fetchFromGitHub
+, lib
+, makeDesktopItem
+, writeScriptBin
+, electronAppName ? "Antimatter Dimensions"
+}:
+
+let
+ dummyElectronApp = ./app.js;
+in
+buildNpmPackage rec {
+  pname = "antimatter-dimensions";
+  version = "unstable-2024-05-11";
```

```suggestion
  version = "0-unstable-2024-05-11";
```

Since there is no tagged version, we should hardcode 0.